TACC/Core-CMS-Custom
# Bug/WP-798: Setup Defaults for Query Client

## Overview
Setup defaults for query client
…

## Related

- [WP-798](https://tacc-main.atlassian.net/browse/WP-798)


## Changes
- Defaults for QueryClientProvider wrapper around Components
## Testing

1. Add the following line to src/client/src/main.tsx:32
```
console.log(queryClient.getQueryCache());
```
2. Navigate to a new page and check in the console if the Query options match the following
`{staleTime: 300000, cacheTime: 600000, refetchOnWindowFocus: false, queryKey: Array(1), queryFn: ƒ, ...}`
## UI
![image](https://github.com/user-attachments/assets/e2adcc0d-21a1-43c4-bbc5-81fe5f5ccedd)

…
